spec/requests/blocked_ip_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe BlockedIp, type: :request do
+  before( :all ) do
+    BlockedIp.destroy_all
+  end
+
+  it "non blocked IP should access page" do
+    get "/ping", headers: { "REMOTE_ADDR" => "192.168.0.1" }
+    expect( response.response_code ).to eq 200
+    expect( response.body ).to include "status"
+  end
+
+  it "blocked IP should not access page" do
+    BlockedIp.create!( ip: "192.168.0.2" )
```

Comment:
We can use [machinist blueprints](https://github.com/inaturalist/inaturalist/blob/main/spec/blueprints.rb) in specs, created with make/make!. If you add a blueprint for BlockedIp, e.g.:
```ruby
BlockedIp.blueprint do
  ip { Faker::Internet.ip_v4_address }
  user { User.make }
end
```
This can just be `blocked_ip = BlockedIp.make!`, and you can use the generated IP value in the header below, `blocked_ip.ip`. You can also send custom values if that's relevant to the spec. As it is now, these specs are failing due to the lack of a specified user, which would come through easily with the blueprint